### PR TITLE
fix: update the volcano metric document.

### PR DESCRIPTION
**Pull request by fengruotj**
fix: the metric doc lacks updates [Issue 3118](https://github.com/volcano-sh/volcano/issues/3118)

### CODE REVIEW

**Comment by fengruotj** [comment]
I've already update the volcano metric document by `pkg/scheduler/metrics/metrics.go`. Are there any other Metrics files I should bring？

File: docs/design/metrics.md [hunk 1]
```diff
@@ -1,39 +1,40 @@
...
-| Metric name | Metric type | Labels | Description |
-| ----------- | ----------- | ------ | ----------- |
-| pod_schedule_errors | Counter |  | The number of kube-batch failed due to an error |
-| pod_schedule_successes | Counter | | The number of kube-batch success in scheduling a job |
-| pod_preemption_victims | Counter | | Number of selected preemption victims |
-| total_preemption_attempts | Counter |  | Total preemption attempts in the cluster till now |
-| unschedule_task_count | Counter | `job`=&lt;job_id&gt; | The number of tasks failed to schedule |
-| unschedule_job_counts | Counter | | The number of job failed to schedule in each iteration |
-| job_retry_counts | Counter | `job`=&lt;job_id&gt; | The number of retry times of one job |
+| **Metric Name**               | **Metric Type** | **Labels**                       | **Description**                                      |
```
Comment (Monokaix) [inline docs/design/metrics.md:30]: Are `pod_schedule_errors` and `pod_schedule_successes` deleted?

File: docs/design/metrics.md [hunk 2]
```diff
@@ -1,39 +1,40 @@
 ## Scheduler Monitoring
 
 ## Introduction
-Currently users can leverage controller logs and job events to monitor scheduler. While useful for debugging, none of this options is particularly practical for monitoring kube-batch behaviour over time. There's also requirement like to monitor kube-batch in one view to resolve critical performance issue in time [#427](https://github.com/kubernetes-sigs/kube-batch/issues/427).
+Currently users can leverage controller logs and job events to monitor scheduler. While useful for debugging, none of this options is particularly practical for monitoring volcano behaviour over time. There's also requirement like to monitor volcano in one view to resolve critical performance issue in time [#427](https://github.com/kubernetes-sigs/volcano/issues/427).
```
Comment (Monokaix) [inline docs/design/metrics.md:4]: Seems the issue link need not to be changed.

Comment (fengruotj) [reply docs/design/metrics.md:4]: Yes, I've already fix it. In addition, I add more metrics defination.

File: docs/design/metrics.md (hunk 1 above)
Comment (fengruotj) [reply docs/design/metrics.md]: I see, I've not found the metrics, like `pod_schedule_errors ` and `pod_schedule_successes `.

**Comment by JesseStutler** [comment]
Could you also add metrics with these PR? #3650 
It adds two counter metrics called `job_completed_phase_count` and `job_failed_phase_count`, recording the information of failed/completed vcjob, if the vc-controller meets a failed/completed vcjob, it will record the metric.

**Comment by fengruotj** [comment]
> Could you also add metrics with these PR? #3650 It adds two counter metrics called `job_completed_phase_count` and `job_failed_phase_count`, recording the information of failed/completed vcjob, if the vc-controller meets a failed/completed vcjob, it will record the metric.

**Comment by fengruotj** [comment]
> Could you also add metrics with these PR? #3650 It adds two counter metrics called `job_completed_phase_count` and `job_failed_phase_count`, recording the information of failed/completed vcjob, if the vc-controller meets a failed/completed vcjob, it will record the metric.

Yes, I've already added it

File: docs/design/metrics.md (hunk 1 above)
Comment (fengruotj) [reply docs/design/metrics.md]: schedule_attempts_total. Maybe, volcano uses the `schedule_attempts_total` counter metrics.

**Comment by Monokaix** [comment]
cc @hwdef @lowang-bh

**Review by lowang-bh — APPROVED** [review]

**Pull request merged**